backend/src/main/kotlin/hu/hollo/news/config/SecurityConfig.kt

```diff
@@ -39,7 +40,8 @@ class SecurityConfig {
             it.contentSecurityPolicy("script-src 'self'")
         }
         .authorizeHttpRequests {
-            it.requestMatchers("/user/login").permitAll()
+            it.requestMatchers("/users/login").permitAll()
+            it.requestMatchers(HttpMethod.GET,"/events").permitAll()
```

Comment:
Itt pontosan minél engedélyezed hogy a /users/login és a /event-re getelhessen mindenki?

Reply:
Ezzel: `it.requestMatchers(HttpMethod.GET,"/events").permitAll()`